references/dump.md

```diff
@@ -0,0 +1,46 @@
+# Dump
+
+The `dump` route allow the creation of database dumps. Dumps are compatible between MeiliSearch versions.
+
+## Trigger dump
+
+<RouteHighlighter method="POST" route="/dumps"/>
+
+Trigger a dump creation process. Once the process is completed, a dump is created in the [dumps folder](/guides/advanced_guides/configuration.md#dumps-folder).
+
+### Example
+
+<code-samples id="post_dump_1" />
+
+#### Response: `202 Accepted`
+
+```json
+{
+  "uid": "20200929-114144097",
+  "status": "processing"
+}
+```
+
+## Get dump status
+
+<RouteHighlighter method="GET" route="/dumps/:dump_uid/status"/>
+
+Get the status of a dump.
+Returned status could be:
+
+- `processing`: Dump creation is not finished.
+- `dump_process_failed`: An error occured during dump process, task aborted.
+- `done`: Dump creation is finished. A dump is created in the [dumps folder](/guides/advanced_guides/configuration.md#dumps-folder)
```

Comment:
```suggestion
- `done`: Dump creation is finished. 

A dump is created in the [dumps folder](/guides/advanced_guides/configuration.md#dumps-folder). After the creation process is over, a dump can be imported to a MeiliSearch Instance at runtime. For more information, follow the [import dumps guide](/guides/advanced_guides/configuration.md#dumps-folder)
```

Reply:
Thats not the guide. Its more like the CLI references. The guide will arrive once everything has been set. So you could remove that part: 
```
For more information, follow the [import dumps guide](/guides/advanced_guides/configuration.md#dumps-folder)
```

Reply:
Done! @bidoubiwa